### PR TITLE
Updates samples for run image extension

### PR DESCRIPTION
**Pull request by natalieparellano**
Branching off of other branch as this is dependent on that fix

### CODE REVIEW

**Comment by natalieparellano** [comment]
Please merge https://github.com/buildpacks/samples/pull/142 first and the diff will be nicer

File: extensions/cowsay/bin/generate [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+RUN curl --version
+
+USER root
+RUN apk update && \
+    apk add --no-cache git perl && \
+    cd /tmp && \
+    git clone https://github.com/schacon/cowsay.git  && \
+    cd cowsay ; ./install.sh /usr/local && \
+    rm -rf /var/cache/apk/* /var/tmp/* /tmp/*
```
Comment (AidanDelaney) [inline extensions/cowsay/bin/generate:20]: Might it be easer to pull in https://pkgs.alpinelinux.org/package/edge/testing/x86/cowsay here?

**Review by AidanDelaney — APPROVED** [review]
I have one nit-pick suggestion.  Other than that, these are the examples we used in the KubeCon talk, and they work well.

**Comment by AidanDelaney** [comment]
Currently blocked on #140

File: extensions/cowsay/bin/generate (hunk 1 above)
Comment (natalieparellano) [reply extensions/cowsay/bin/generate:20]: Thanks!

**Pull request merged**